lib/gis_robot_suite.rb

```diff
@@ -57,14 +57,28 @@ def self.vector?(cocina_object)
     media_type(cocina_object) == 'application/x-esri-shapefile'
   end
 
+  RASTER_TYPES = %w[image/tiff].freeze
+
   def self.raster?(cocina_object)
-    %w[image/tiff application/x-ogc-aig].include? media_type(cocina_object)
+    raise "#{cocina_object.externalIdentifier} is ArcGrid format: 'application/x-ogc-aig'" if media_type(cocina_object) == 'application/x-ogc-aig'
+
+    RASTER_TYPES.include?(media_type(cocina_object))
   end
 
   def self.media_type(cocina_object)
     geographic_form(cocina_object, 'media type')
   end
 
+  def self.layertype(cocina_object)
+    if vector?(cocina_object)
```

Comment:
I don't see that we are requesting a layertype for vectors in the code, but left it in here.